Dockerfile.ci

```diff
@@ -340,6 +308,23 @@ RUN if [[ -n "${ADDITIONAL_PYTHON_DEPS}" ]]; then \
         pip install ${ADDITIONAL_PYTHON_DEPS}; \
     fi
 
+RUN \
+    export AWSCLI_IMAGE="amazon/aws-cli:latest" && \
+    export AZURECLI_IMAGE="mcr.microsoft.com/azure-cli:latest" && \
+    export GCLOUD_IMAGE="gcr.io/google.com/cloudsdktool/cloud-sdk:latest" && \
+    echo -e "\
+    alias aws=\"docker run --rm -it -v /root/.aws:/root/.aws ${AWSCLI_IMAGE}\" \n\
```

Comment:
This volume mount is wrong. Since it is using the host's docker daemon, this will mount from /root/.aws on the host, not what /root/.aws is mounted to in this running image.

Reply:
It will work OK. I tested it. There are two ways of running breeze ( I described it with TESTING.rst)

1) if you run it as normal you would mount the /root/.aws from the container which means that you have to authenticate separately and this authentication will not survive restart of Breeze.

2) if you run `breeze --forward-credentials` it will mount your "${HOME}/.aws" to the breeze container's /root/.aws and from there it will be further mounted to the "aws" image. Which means that you can use your host credentials and they will survive Breeze restart.

Reply:
Comment from BREEZE.rst

```
For each of those CLI you need to provide credentials. You can do it in one of the ways:

* You can forward credentials from your host machine when starting breeze with
  ``--forward-credentials`` flag added when you start ``Breeze``. Just a watch-out - this makes
  your host credentials available inside the image, so use it with care.

* If you do not want to forward credentials, you can use traditional authentication methods provided by
  each of the tools (for example ``gcloud auth login``) - the credentials are stored in a configuration
  directory specific for each CLI tool until you stop breeze session. You will need to re-authenticate
  when you restart ``Breeze``.
```

Reply:
> If you run `breeze --forward-credentials` it will mount your "${HOME}/.aws" to the breeze container's /root/.aws and from there it will be further mounted to the "aws" image. This means that you can use your host credentials and they will survive Breeze restart.

Is this docker-in-docker, or mounting /var/run/docker.sock from the host.

If it's the latter: this will not work. All volume mounts are resolved from the host:


```
airflow ❯ touch ~/.aws/example
airflow ❯ docker run -ti -v /var/run/docker.sock:/var/run/docker.sock -v $HOME/.aws:/root/.aws docker:latest sh
/ # echo container 1 -- "breeze"
container 1 -- breeze
/ # ls -altr /root/.aws/example 
-rw-r--r--    1 1000     1000             0 Jun  4 13:51 /root/.aws/example
/ # hostname
4b4a523958cb
/ # docker run -ti -v /var/run/docker.sock:/var/run/docker.sock -v /root/.aws/.aws:/root/.aws docker:latest sh
/ # echo container 2 - cloud tools
container 2 - cloud tools
/ # hostname
dcae57c09ea4
/ # ls -altr /root/.aws
total 18
drwxr-xr-x    2 root     root             2 Jun  4 13:55 .
drwx------    3 root     root             4 Jun  4 13:55 ..
```

Reply:
I think with that, you saved me few hours of debugging of someone's problem :)? Thanks @ashb !

Reply:
I updated the clis (I had to change them from aliases into /usr/bin/script to take into account HOST_HOME variable. But it works like a charm now even without --forward-credentials. Documentation updated as well..